src/linearmixedmodel.jl

```diff
@@ -312,7 +312,7 @@ diagonal blocks from the conditional variance-covariance matrix,
     s² Λ(Λ'Z'ZΛ + I)⁻¹Λ'
 """
 function condVar(m::LinearMixedModel{T}) where {T}
-    L = m.L
+    return [condVar(m, fnm) for fnm in fnames(m)]
     s = sdest(m)
     @static if VERSION < v"1.6.1"
```

Comment:
I'm tempted to up the julia compat to 1.6.1 for the whole package so that we can drop this branch.